README.md

```diff
@@ -31,15 +31,14 @@ The reverse proxy using Nginx is built like this:
 
     docker build -t reverseproxy docker-nginx/
 
-The CodeScene image should already be available from [Docker Hub](https://hub.docker.com/r/empear/ubuntu-onprem/) under
-`empear/ubuntu-onprem:latest`, but can also be built locally like this:
+The CodeScene image is available from [Docker Hub](https://hub.docker.com/r/empear/ubuntu-onprem/).
 
-    docker build -t empear/ubuntu-onprem docker-codescene/
+If you want to use a specific version of CodeScene, you can add a tag in the `docker-compose.yml`:
 
-If you want to use a specific version of CodeScene, you can add a `--build-arg`:
+```
+    image: empear/ubuntu-onprem:latest
+```
 
-    docker build  --build-arg CODESCENE_VERSION=3.X.Y -t empear/ubuntu-onprem docker-codescene/
-	
 ## Run
```

Comment:
There's a link in the README to https://github.com/empear-analytics/docker-codescene-nginx-self-signed-ssl/blob/master/docker-codescene/codescene which no longer exists. 
There are likely a few other broken links and source code listings.

Reply:
I tried to update the README, but apparently missed this reference. Thx